chapters/annotations.tex

```diff
@@ -1473,24 +1523,28 @@ \section{Graphical User Interface}\label{annotations-for-the-graphical-user-inte
 Additionally, when a user defined model is structurally inconsistent, a tool should try to pinpoint in which class the error is present.
 This annotation avoids then to show a wrong error message.
```

Comment:
I think we might clarify this in the future - I realize it wasn't changed, but I think that clarifying this would give insights in how to write the one-liner. But perhaps best to have it as a separate later PR.

Reply:
Sounds good.